cluster-scope/base/rbac.authorization.k8s.io/clusterroles/sre-read-cr/clusterrole.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sre-group-read
+rules:
+- apiGroups:
+  - ""
```

Comment:
@dystewart seems like the issue you are aiming to resolve is only asking for asking to this [resource](https://docs.openshift.com/container-platform/4.8/rest_api/metadata_apis/apirequestcount-apiserver-openshift-io-v1.html), and not all of the core group (denoted by `""`), can you scope this to just that resource? you will need to update `apiGroups` and `resources` field. You should also rename the name of this role/rolebinding to something like `apirequestcount-reader`